examples/data-objects/presence-tracker/src/FocusTracker.ts

```diff
@@ -107,7 +107,7 @@ export class FocusTracker extends TypedEventEmitter<IFocusTrackerEvents> {
     }
 
     /**
-     * Get a copy of the internal mouse position presences map
+     * Get a copy of the internal focus status map
```

Comment:
This doesn't seem to be returning a map at all though, probably needs more updating.

Reply:
Whoops!